auto-generated/api-testing/vfadd.c

```diff
@@ -244,3 +244,243 @@ vfloat64m8_t test_vfadd_vf_f64m8_m(vbool8_t mask, vfloat64m8_t op1, float64_t op
   return __riscv_vfadd_vf_f64m8_m(mask, op1, op2, vl);
 }
 
+vfloat16mf4_t test_vfadd_vv_f16mf4_rm(vfloat16mf4_t op1, vfloat16mf4_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16mf4_rm(op1, op2, __RISCV_FRM_RNE, vl);
```

Comment:
Are the __RISCV_FRM_RNE and other values documented? The web view makes it hard to find if it is.

Reply:
Changes within this repo generally will break into at least two part: 1) generator/doc update and 2) re-run generator.

So personally I will switch to the `commit` tab and found those doc change for easier review :)

e.g.
https://github.com/riscv-non-isa/rvv-intrinsic-doc/pull/226/commits/f1ad3258b9e7de52f57c811cf90d3b5541604f94